src/gui/Sidebar.tsx

```diff
@@ -173,18 +171,8 @@ export class ReviewQueueListView extends ItemView {
         }
 
         folderTitleEl.onClickEvent(() => {
-            for (const child of childrenEl.childNodes as NodeListOf<HTMLElement>) {
-                if (child.style.display === "block" || child.style.display === "") {
-                    child.style.display = "none";
-                    (collapseIconEl.childNodes[0] as HTMLElement).style.transform =
-                        "rotate(-90deg)";
-                    deck.activeFolders.delete(folderTitle);
```

Comment:
What is this for?

Reply:
this line: deck.activeFolders.delete(folderTitle);

Reply:
This stores the collapsed/expanded state of each folder (folder names being things like
    // "TODAY", "NEW" or formatted dates).

So here where the tree node is being collapsed, the folder name is being removed

Reply:
Is the call for this function at this spot correct it seems for me not necessary?

Reply:
Like me, do you also think that the `activeFolders` is used to maintain the state of which folders are expanded?
This code is being executed when the user clicks on an expanded folder name, causing it to collapse.
So it seems to me that calling `deck.activeFolders.delete(folderTitle);` is valid.

Reply:
Yeah I think your right I will fix this